configs/_base_/models/paconv_cuda_ssg.py

```diff
@@ -0,0 +1,7 @@
+_base_ = './paconv_ssg.py'
+
+model = dict(
+    backbone=dict(
+        sa_cfg=dict(
+            type='PAConvSAModuleCUDA',
```

Comment:
CUDAPAConvSAModule might be better

Reply:
Do you think `PAConvCUDASAModule` is better? Since we use `PAConv` and `PAConvCUDA` for these two basic conv operation.

Reply:
Change to `PAConvCUDASAModule`.